README.md

```diff
@@ -184,6 +184,33 @@ $ ./pgremapper export-mappings <osdspec> ... [--output <file>] [--whole-pg]
 * `--output`: Write output to the given file path instead of `stdout`.
 * `--whole-pg`: Export all mappings for any PGs that include the given OSD(s), not just the portions pertaining to those OSDs.
 
+### export-crush-change-mappings
```

Comment:
After thinking about this, instead of `export` I think I might prefer `generate` in this command name.

Reply:
Makes sense. Updated!